cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/Filters/FilterCollection/FilterShowlistInput/index.tsx

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IconButton, WithStyles } from '@material-ui/core';
```

Comment:
I think I mentioned this in other PRs, if not please use individual sub-modules from material-ui.

```
import IconButton from '@material-ui/core/IconButton'
import { WithStyles } from '@material-ui/core/styles/withStyles';
```

Reply:
Adding as an umbrella comment for all the imports.